fieldmaptrack/fieldmap.py

```diff
@@ -230,7 +230,14 @@ def read_fieldmap(self, fname):
         self.rz_step = (self.rz_max - self.rz_min) / (self.rz_nrpts - 1.0) \
             if self.rz_nrpts > 1 else 0.0
         self.rx_zero = np.where(self.rx == 0)[0][0]
-        self.ry_zero = np.where(self.ry == 0)[0][0]
+        try:
+            self.ry_zero = np.where(self.ry == 0)[0][0]
```

Comment:
I think you could use the opportunity to optimize the new code and also the original code just below (lines 241-248):
```python3
self.ry_zero = np.argmin(np.abs(np.asarray(self.ry)))
if self.ry[self.ry_zero] != 0.0:
    print('data set does not contain y=0 !')
```
As long as you test it with the current ID fieldmap in analysis I think it is ok.

Reply:
Implemented and tested!